### ISSUE TITLE
Problem installing Grid-GK5 on homepage

### ISSUE THREAD

**Issue by GielPieren** [GavickPro/Grid-GK5#21]
I have problem installing the grid module. After following the documentation the grid positions don't apear in the module positions menu. I'am using JF connecto template with Gantry framework, does that have to do with my problem?

**Comment by dziudek** [comment]
Hello,

Please add grid1, grid2 etc. module positions in your templateDetails.xml file.

**Comment by GielPieren** [comment]
how?

**Comment by dziudek** [comment]
Please read step #1: https://www.gavick.com/documentation/joomla-templates/customizing-joomla-template/how-to-add-new-module-position

**Comment by GielPieren** [comment]
Dear mister Dziuda,

Step 1 i allready did, i found it on another website as well.
The module positions i can find now in my template, but they still don't
work. If i want to try step 3 i don't know wich file i have to change
because i don't have a GK template and there are more default files.

yours sincerly,

Giel Pieren

2015-04-24 20:58 GMT+02:00 Tomasz Dziuda notifications@github.com:

**Comment by dziudek** [comment]
But what still doesn't work? You still cannot use i.e. grid1 module position in the module?

**Comment by GielPieren** [comment]
The Grid is showing in the template as a module position, but it isn't
working as a module position. it continue to say : _error:_ Specified
module position doesn't exist or is blank! and the grid1 module postion is
only one bar, or doesn't apear at all.

[image: Inline afbeelding 2]

view image:
It only appears in the top region of the module and the rest is blank.

2015-04-26 9:37 GMT+02:00 Tomasz Dziuda notifications@github.com:

**Comment by dziudek** [comment]
Please reupload the image.

Did you set some module on the newly added module position?

**Comment by GielPieren** [comment]
yes, i connect the grid with one of the main postions of my template. The
image that is partly shown i have connected with my custom html module
myGrid1 and also a custom html module connected with MyGrid2 but there is
nothing to see at all.
like this:
[image: Inline afbeelding 1]

2015-04-30 18:45 GMT+02:00 Tomasz Dziuda notifications@github.com:

**Comment by dziudek** [comment]
Please reply on github directly, because your images from e-mail are not visible.
